website/static/js/share/results.js

```diff
@@ -151,8 +151,18 @@ var Contributor = {
             m('a', {
                 href: '#',
                 onclick: function() {
-                    utils.updateFilter(vm, 'match:contributors.familyName:' + contributor.familyName, true);
-                    utils.updateFilter(vm, 'match:contributors.givenName:' + contributor.givenName, true);
+                    var glen = contributor.givenName ? contributor.givenName.length : 0;
```

Comment:
I think the variables should have more descriptive names such as`givenNameLength` or even `givenLength`